src/sonic-yang-models/yang-templates/sonic-acl.yang.j2

```diff
@@ -144,7 +152,7 @@ module sonic-acl {
 
 				leaf ETHER_TYPE {
 					type string {
-						pattern "(0x88CC|0x8100|0x8915|0x0806|0x0800|0x86DD|0x8847)";
+					    pattern "0x[6-9a-fA-F][0-9a-fA-F]{2}|0x[1-9a-fA-F][0-9a-fA-F]{3}";
```

Comment:
0x0800 was acceptable, but not now.

Reply:
Added only those 2 to pattern to maintain backwards compatibility